src/test/script_p2sh_tests.cpp

```diff
@@ -18,6 +18,11 @@
 #include <boost/test/unit_test.hpp>
 
 // Helpers:
+bool IsStandardTx(const CTransaction& tx, std::string& reason)
```

Comment:
In commit "Remove ::IsStandardTx(tx, reason) alias" (fa8a7f01fe1b6db98097021276ed5d929faadbec)

Would be good to declare this static to make sure nobody outside this file can call this

Reply:
Good point. Will do on the next push or as a follow-up.

Reply:
Fixed in #25772